docs/tutorials/regression.ipynb

```diff
@@ -2,204 +2,391 @@
  "cells": [
   {
    "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
    "source": [
     "# Regression\n",
     "\n",
     "This tutorial uses safeds on **house sales data** to predict house prices.\n"
-   ],
-   "metadata": {
-    "collapsed": false
-   }
+   ]
   },
   {
    "cell_type": "markdown",
-   "source": [
-    "1. Load your data into a `Table`, the data is available under `docs/tutorials/data/pricing.csv`:\n"
-   ],
    "metadata": {
     "collapsed": false
-   }
+   },
+   "source": [
+    "## File and Imports\n",
+    "\n",
+    "Start by creating a Python-File with the suffix ``.py``.\n",
+    "\n",
+    "Import the classes you want to use from safeds.\n"
```

Comment:
```suggestion
    "Import the classes you want to use from Safe-DS.\n"
```

Reply:
Which classes would that? Since later cells include the required imports, I'd rather remove this sentence here altogether.